### PR TITLE
refactor(argentina): content updates

### PR DESCRIPTION
**Pull request by goldpbear**

### CODE REVIEW

File: src/flavors/argentina/config.json [hunk 1]
```diff
@@ -61,12 +61,53 @@
         "type": "raster-tile",
         "url": "https://{s}.tile.opentopomap.org/{z}/{x}/{y}/256/png8?app_id=tFZyfnyJAmhfh5gdoGcR&app_code=vJ8o9OCQ1o0Y2wwbRspzSA&lg=eng"
       },
+      {
+        "id": "argentina-campanas",
+        "type": "vector-tile",
+        "url": "https://vector-tiles.mapseed.org/argentina-campanas/{z}/{x}/{y}.pbf",
+        "source_layer": "argentina-campanas",
+        "mapbox_layers": [
```
Comment (goldpbear) [inline src/flavors/argentina/config.json]: @Brian393 -- note that here you can dial in whatever mapbox layers you want from the source data in the `argentina-campanas` vector tiles. So you could add fills, change the outline styling, etc.

Comment (Brian393) [reply src/flavors/argentina/config.json]: @goldpbear 

But the question is, how do I reach that source data? When I follow the link, https://vector-tiles.mapseed.org/argentina-campanas/{z}/{x}/{y}.pbf, it takes me to an empty xml file that says something about a key....

Comment (goldpbear) [reply src/flavors/argentina/config.json]: The full MapboxGL style spec is here: https://www.mapbox.com/mapbox-gl-js/style-spec/.

There's a lot you can do!

**Comment by Brian393** [comment]
Thanks, that's brilliant, I totally see how this thing works. The API
editor is also great.

I am about to send a PR which makes the polygons transparent and sets the
default visibility of the right sidebar to false. I will work on filling in
the comment windows for the five campaigns, then reactivate the right
sidebar later.

thanks a million, Brian

**Pull request merged**